pulpcore/cli/openapi.py

```diff
@@ -32,7 +32,8 @@ def __init__(
             "Accept": "application/json",
         }
         self._session = requests.session()
-        self._session.auth = (username, password)
+        if username or password:
```

Comment:
```suggestion
        if username and password:
```

Reply:
My only concern here is that this will silence errors in situations where a user accidentally defines username or password but not both. wdyt?

Reply:
Then we need more if/else.
If either of them is None and the other one isn't, MyPy complains.

Reply:
In that case, I can switch it to `and`. We should probably add this validation elsewhere.